playbooks/generic-enroll_caasp_nodes_in_caasp_cluster.yml

```diff
@@ -47,3 +50,5 @@
       args:
         executable: /bin/bash
         chdir: "{{ upstream_repos_clone_folder }}/ccp/automation/velum-bootstrap"
+      tags:
+        - skip_ansible_lint
```

Comment:
why?

Reply:
Because we don't have enough information on the changed_when: condition.
It also needs to be shell vs command, because velum tooling didn't handle different shell IIRC. All of that together needs a skip_ansible_lint.

Reply:
I have now added, to all the tasks that require a skip_ansible_lint, the reason of the ansible-lint skip.